### ISSUE TITLE
Age of onset: any age

### ISSUE THREAD

**Issue by PabloBotas** [obophenotype/human-phenotype-ontology#6791]
**Preferred term label:**
All age

**Synonyms**
Any age

**Definition (free text, please give PubMed ID)**
Onset of disease manifestations occur at any age.

For Orphanet's definition, see: http://www.orphadata.org/cgi-bin/img/PDF/OrphadataFreeAccessProductsDescription.pdf.

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0003674

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
This is just doing a quick search on Orphanet's XML data after filtering for disorderGroup = disorder and forcing them to have symptoms associated to them (total of 3649 entities considered). I can repeat this without filtering. There is no guarantee that each entity is ONLY associated with this age of onset.

List of 380 terms: `ORPHA:69061, ORPHA:90068, ORPHA:448237, ORPHA:330015, ORPHA:284984, ORPHA:2260, ORPHA:231, ORPHA:2201, ORPHA:36412, ORPHA:469, ORPHA:100093, ORPHA:2309, ORPHA:108, ORPHA:2238, ORPHA:781, ORPHA:352470, ORPHA:493, ORPHA:319251, ORPHA:31824, ORPHA:558, ORPHA:229, ORPHA:232, ORPHA:29207, ORPHA:805, ORPHA:3193, ORPHA:652, ORPHA:357154, ORPHA:163, ORPHA:1572, ORPHA:79233, ORPHA:51083, ORPHA:288, ORPHA:23, ORPHA:33001, ORPHA:2978, ORPHA:622, ORPHA:447, ORPHA:3299, ORPHA:543, ORPHA:251636, ORPHA:3129, ORPHA:191, ORPHA:84090, ORPHA:37202, ORPHA:26, ORPHA:247525, ORPHA:101, ORPHA:157850, ORPHA:548, ORPHA:29822, ORPHA:182, ORPHA:799, ORPHA:99013, ORPHA:86814, ORPHA:73247, ORPHA:2552, ORPHA:231111, ORPHA:211067, ORPHA:3002, ORPHA:139491, ORPHA:774, ORPHA:79455, ORPHA:75249, ORPHA:147, ORPHA:79095, ORPHA:890, ORPHA:58, ORPHA:217260, ORPHA:98969, ORPHA:930, ORPHA:3238, ORPHA:1160, ORPHA:1656, ORPHA:700, ORPHA:757, ORPHA:52, ORPHA:300385, ORPHA:90003, ORPHA:31204, ORPHA:95613, ORPHA:247257, ORPHA:31112, ORPHA:701, ORPHA:228277, ORPHA:137599, ORPHA:227982, ORPHA:70, ORPHA:18, ORPHA:981, ORPHA:810, ORPHA:772, ORPHA:98762, ORPHA:93109, ORPHA:3385, ORPHA:63455, ORPHA:2024, ORPHA:39812, ORPHA:3392, ORPHA:2084, ORPHA:225, ORPHA:1163, ORPHA:3111, ORPHA:90026, ORPHA:549, ORPHA:2302, ORPHA:319213, ORPHA:589, ORPHA:2686, ORPHA:91138, ORPHA:396, ORPHA:464453, ORPHA:26791, ORPHA:2149, ORPHA:327, ORPHA:36234, ORPHA:83597, ORPHA:45, ORPHA:42062, ORPHA:99827, ORPHA:171612, ORPHA:616, ORPHA:90042, ORPHA:329, ORPHA:98759, ORPHA:2285, ORPHA:3347, ORPHA:2406, ORPHA:425, ORPHA:83601, ORPHA:508410, ORPHA:163703, ORPHA:351, ORPHA:822, ORPHA:66529, ORPHA:231625, ORPHA:1267, ORPHA:95427, ORPHA:845, ORPHA:729, ORPHA:93110, ORPHA:53721, ORPHA:3437, ORPHA:98827, ORPHA:231580, ORPHA:99824, ORPHA:1546, ORPHA:1304, ORPHA:1306, ORPHA:94090, ORPHA:449280, ORPHA:140896, ORPHA:231445, ORPHA:2255, ORPHA:83452, ORPHA:231632, ORPHA:64755, ORPHA:90797, ORPHA:86884, ORPHA:131, ORPHA:157, ORPHA:90362, ORPHA:3386, ORPHA:251915, ORPHA:2048, ORPHA:97282, ORPHA:926, ORPHA:79277, ORPHA:2897, ORPHA:731, ORPHA:66661, ORPHA:903, ORPHA:1202, ORPHA:95409, ORPHA:702, ORPHA:405, ORPHA:723, ORPHA:319218, ORPHA:101016, ORPHA:758, ORPHA:666, ORPHA:83313, ORPHA:2237, ORPHA:8, ORPHA:99826, ORPHA:889, ORPHA:3467, ORPHA:2764, ORPHA:2035, ORPHA:31202, ORPHA:54057, ORPHA:2582, ORPHA:221061, ORPHA:352447, ORPHA:79445, ORPHA:75563, ORPHA:3325, ORPHA:90000, ORPHA:423, ORPHA:73267, ORPHA:60, ORPHA:50251, ORPHA:251643, ORPHA:99829, ORPHA:449285, ORPHA:31826, ORPHA:863, ORPHA:71213, ORPHA:94093, ORPHA:221, ORPHA:69078, ORPHA:216694, ORPHA:454836, ORPHA:137817, ORPHA:882, ORPHA:3204, ORPHA:95455, ORPHA:68, ORPHA:98870, ORPHA:507, ORPHA:1330, ORPHA:664, ORPHA:65682, ORPHA:139402, ORPHA:703, ORPHA:63442, ORPHA:228119, ORPHA:33475, ORPHA:201, ORPHA:1879, ORPHA:1653, ORPHA:20, ORPHA:728, ORPHA:293807, ORPHA:1930, ORPHA:289504, ORPHA:707, ORPHA:163934, ORPHA:189427, ORPHA:274, ORPHA:436, ORPHA:214, ORPHA:212, ORPHA:300493, ORPHA:857, ORPHA:280679, ORPHA:1772, ORPHA:79148, ORPHA:99828, ORPHA:2134, ORPHA:2086, ORPHA:231160, ORPHA:355, ORPHA:767, ORPHA:73, ORPHA:97279, ORPHA:31205, ORPHA:3389, ORPHA:324625, ORPHA:71211, ORPHA:43, ORPHA:98755, ORPHA:2070, ORPHA:98916, ORPHA:254886, ORPHA:869, ORPHA:367, ORPHA:99745, ORPHA:84085, ORPHA:86893, ORPHA:228123, ORPHA:2116, ORPHA:85138, ORPHA:6, ORPHA:306682, ORPHA:524, ORPHA:84142, ORPHA:533, ORPHA:416, ORPHA:145, ORPHA:360, ORPHA:99429, ORPHA:101111, ORPHA:846, ORPHA:297, ORPHA:91, ORPHA:764, ORPHA:499, ORPHA:51890, ORPHA:838, ORPHA:91547, ORPHA:90036, ORPHA:466677, ORPHA:646, ORPHA:90033, ORPHA:46488, ORPHA:422, ORPHA:2609, ORPHA:67, ORPHA:328, ORPHA:2573, ORPHA:509, ORPHA:90037, ORPHA:91546, ORPHA:579, ORPHA:411703, ORPHA:79139, ORPHA:910, ORPHA:325, ORPHA:65683, ORPHA:83317, ORPHA:3261, ORPHA:79457, ORPHA:1552, ORPHA:722, ORPHA:79135, ORPHA:331, ORPHA:140989, ORPHA:1916, ORPHA:408, ORPHA:91350, ORPHA:637, ORPHA:67043, ORPHA:90289, ORPHA:47159, ORPHA:419, ORPHA:770, ORPHA:927, ORPHA:96, ORPHA:160, ORPHA:1880, ORPHA:36237, ORPHA:2604, ORPHA:35909, ORPHA:2126, ORPHA:302, ORPHA:340, ORPHA:31825, ORPHA:567, ORPHA:173, ORPHA:650, ORPHA:3318, ORPHA:33208, ORPHA:53035, ORPHA:530, ORPHA:348, ORPHA:225147, ORPHA:326, ORPHA:97230, ORPHA:673, ORPHA:391, ORPHA:46724, ORPHA:424, ORPHA:285, ORPHA:99825, ORPHA:330021, ORPHA:786, ORPHA:1549, ORPHA:1885, ORPHA:98756, ORPHA:2583, ORPHA:1560, ORPHA:335, ORPHA:107, ORPHA:2495, ORPHA:2485, ORPHA:154, ORPHA:73263, ORPHA:60033, ORPHA:1727, ORPHA:586, ORPHA:2807, ORPHA:254478, ORPHA:90044, ORPHA:189439, ORPHA:74, ORPHA:79086, ORPHA:198, ORPHA:724, ORPHA:43116, ORPHA:54595`

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-7090-9648

**Additional comments**
1. A similar term should be added under Age of Death: https://hpo.jax.org/app/browse/term/HP:0011420
2. This should be applied to diseases, not individual patients. It should not refer to a temporal pattern, but simply to an age of onset/death.

Thanks!
Pablo

**Comment by pnrobinson** [comment]
Hi Pablo, there are actually very few features that truly occur with a uniform distribution. We have been annotating cases with variable onsets like this
infantile onset 2/7
childhood onset 5/7

This will allow us to better use this information algorithmically. It would be good to look at a few of these examples in detail and figure out what is going on, but I doubt that we ever have a distribution like p_n =1/n where n=100 years and p_n is the probability of onset in year n

**Comment by drseb** [comment]
I like the idea from pablo. Saying that the onset can be at all ages does not mean it is a uniform distribution. If that is clearly defined, then this information adds value without introducing any noise

**Comment by PabloBotas** [comment]
Hi,
Yes, this is my point. It doesn't necessarily imply uniformity across all ages. But Peter's point still holds nevertheless. This is coming from looking at Orphanet's xmls and seeing that it is currently being used, so, it's better to offer a standard term.

**Comment by drseb** [comment]
Which point from Peter?

**Comment by PabloBotas** [comment]
That it may seem to imply a uniform distribution across ages

**Comment by drseb** [comment]
So the issue is orphanets definition: „ From birth  to  adulthood  without  peak  of  onset“, which implies uniformity?

**Comment by PabloBotas** [comment]
I don't think there is an issue with the definition, which is completely valid. I think probably not all of those Orphanet's disorders really should be annotated with such a broad term.

My read of Peter's point was not really about the term definition, but about whether or not that really definition applies often and the worry of people defaulting to using it when the reality is that the lack of signal is due to lack of a homogeneous cohort or lack of data.

Having said this: having such term makes sense from a completeness point of view

**Comment by pnrobinson** [comment]
My main issue is that if we annotate a disease as "arbitrary onset" then we cannot learn from the data. It is better to annotate what is actually reported. If we continue to get more and more data, then at some point we can have a distribution of onsets across each disease. If on the other hand, we see that some phenotype has 2/10 infantile, 5/10 childhood and 3/10 juvenile onset in a study and we annotate it as 10/10 arbitrary onset, then we will never be able to combine this data with data from a paper published in 3 years with say 1/10 infantile, 7/10 childhood and 2/10 juvenile onset  where slowly the true distribution becomes apparent. 


If there is not a uniform distribution, then we really should not be using a term like "Onset of disease manifestations occur at any age." because we are discarding data.

**Comment by pnrobinson** [comment]
@PabloBotas can you provide one or two examples of where it would be appropriate to use the term? Are there uniform distributions?

**Comment by PabloBotas** [comment]
Annotating pretty much any infectious disease... maybe with the exception big young children. COVID for instance

**Comment by pnrobinson** [comment]
Well, there are important age differences even for infectious diseases. See for instance Figure 1a of this paper
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6890770/
If you want to denote "I do not know when the onset is", then it is best not to annotate onset, but if you have any data at all, it is best to record it. Otherwise we will not be able to learn from data.

**Comment by PabloBotas** [comment]
Correct such term should not be used when you do not know.

If you think that the term is not realistically applicable, then by all means, it should not be included. Of course it's better to annotate as precise as possible and the best way is to have anonymous individual (onset) data. For this situation this term would not apply.

Your call ;)

**Comment by PabloBotas** [comment]
Fair enough. I think you have good reasons not to do it. There is the risk of not having people annotating enough, but hopefully that won't be the case. Thanks!
